apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -825,6 +823,17 @@ export default function teacherSections(state = initialState, action) {
       }
     }
 
+    // PL Sections must use email logins and its grade value should be "pl"
+    const loginTypeSettings = {};
+    const gradeSettings = {};
+    if (
+      action.props.participantType &&
+      action.props.participantType !== ParticipantAudience.student
+    ) {
+      loginTypeSettings.loginType = SectionLoginType.email;
+      gradeSettings.grade = PlGradeValue;
+    }
```

Comment:
If you're in section.rb again in the the near future, it might be useful to add a comment to the validation logic indicating that this is why we chose this value.

Reply:
Done

Reply:
I think this is another example of where (to James's recurring point) we should have RED's database be separate from prod's with a translation layer in between. then, RED could write their own logic to populate this field if they need it.